src/Sulu/Component/Content/Metadata/Loader/schema/form-1.0.xsd

```diff
@@ -16,11 +16,12 @@
     <xs:element name="form" type="formType"/>
 
     <xs:complexType name="formType">
-        <xs:all>
+        <xs:choice maxOccurs="unbounded">
```

Comment:
What was that change for?

Reply:
it is copied from `template-1.0.xsd` it allows to add the mulitple tags to the form